src/ibex_bluesky_core/devices/reflectometry/__init__.py

```diff
@@ -0,0 +1,106 @@
+"""Devices specific to Reflectometry beamlines."""
+
+import asyncio
+
+from ophyd_async.core import (
+    AsyncStatus,
+    SignalR,
+    SignalW,
+    StandardReadable,
+    StandardReadableFormat,
+    observe_value,
+)
+from ophyd_async.epics.core import epics_signal_r, epics_signal_w
+
+from ibex_bluesky_core.utils import get_pv_prefix
+
+
+class ReflParameter(StandardReadable):
+    """Utility device for a reflectometry server parameter."""
+
+    def __init__(self, prefix: str, name: str, changing_timeout: float) -> None:
+        """Reflectometry server parameter.
+
+        Args:
+            prefix: the PV prefix.
+            name: the name of the parameter.
+            changing_timeout: time to wait for the CHANGING signal to go to False after a set.
+
+        """
+        with self.add_children_as_readables(StandardReadableFormat.HINTED_SIGNAL):
+            self.readback: SignalR[float] = epics_signal_r(float, f"{prefix}REFL_01:PARAM:{name}")
+        self.setpoint: SignalW[float] = epics_signal_w(float, f"{prefix}REFL_01:PARAM:{name}:SP")
+        self.changing: SignalR[bool] = epics_signal_r(
+            bool, f"{prefix}REFL_01:PARAM:{name}:CHANGING"
+        )
+        self.redefine = ReflParameterRedefine(
+            prefix=f"{prefix}REFL_01:PARAM:{name}:", name=name + "redefine"
```

Comment:
Why do we need an explicit name here? Won't it get renamed by ophyd-async anyway?